cpu/atmega_common/cpu.c

```diff
@@ -83,7 +92,7 @@ void cpu_init(void)
     /* set the Division factor to 1 results in divisor 2 for internal Oscillator
```

Comment:
```CLKPR = (1 << CLKPCE);``` and ```CLKPR = 0;``` can be removed as these are the exact operations used by ```atmega_set_prescaler()``` in ```boards/common/atmega/board.c```

I think when I submitted the PR for ```atmega_set_prescaler()``` I just didn't notice that ATmega256rfr2 already did this.

Reply:
Hmmm... I think you're right. The assumption is that the board manages the clocking, which can only be configured at board level and not at the cpu level.

I'll remove the lines and test.